cdap-spark-core-base/src/main/java/io/cdap/cdap/app/runtime/spark/SparkProgramRuntimeProvider.java

```diff
@@ -289,8 +294,10 @@ private ProgramRunner createSparkProgramRunner(Injector injector,
   /**
    * Returns an array of {@link URL} being used by the {@link ClassLoader} of this {@link Class}.
    */
-  private synchronized SparkRunnerClassLoader createClassLoader(boolean filterScalaClasses,
-                                                                boolean rewriteYarnClient) throws IOException {
+  private synchronized SparkRunnerClassLoader createClassLoader(
+    boolean filterScalaClasses,
```

Comment:
Generally I prefer alignment like this:

```
private synchronized SparkRunnerClassLoader createClassLoader(boolean filterScalaClasses,
                                                              boolean rewriteYarnClient, 
                                                              boolean rewriteCheckpointFileName) throws IOException {
  // Code
}
```

As it gives a clearer separation between method signature and the method body.

Reply:
If the last variable is too long, it's ok to use a shorter name since it is only local to the method.